README.md

```diff
@@ -1,56 +1,48 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Book Store
 
-# Project Name
-
-> Description the project.
+## Screenshot ![see](./screenshot.png?raw=true 'Title')
 
+> The app retrieves available tickers from the API lists them in an home page. When a particular ticker is clicker it routes to the information of that ticker.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+- React
 
-[Live Demo Link](https://livedemo.com)
+## Live Demo
 
+[Live Demo Link](https://mnekx.github.io/crypto-tickers/)
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
 ### Setup
 
+1. git clone https://github.com/mnekx/crypto-tickers.git
+2. cd crypto-tickers
+3. npm install
+4. npm start to serve into local 3000 port
+
 ### Install
 
-### Usage
+1. git clone https://github.com/mnekx/crypto-tickers.git
+2. cd crypto-tickers
+3. npm install
+4. npm start to serve into local 3000 port
 
 ### Run tests
 
-### Deployment
+npm run test
 
+### Deployment
 
+https://mnekx.github.io/crypto-tickers/
 
```

Comment:
- [ ] The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Please you have to add it